models/public/octave-resnet-101-0.125/octave-resnet-101-0.125.md

```diff
@@ -16,30 +15,30 @@ The `octave-resnet-101-0.125` model is a modification of [ResNet-101](https://ar
 
 ## Accuracy
 
-| Metric | Value |
-| ------ | ----- |
+| Metric | Value  |
+| ------ | ------ |
 | Top 1  | 79.182%|
-| Top 5  | 94.42%|
+| Top 5  | 94.42% |
 
 ## Input
 
-A blob that consists of a single image of `1x3x224x224` in `RGB` order. Before passing the image blob into the network, subtract RGB mean values as follows: [124,117,104]. In addition, values must be divided by 0.0167.
+A blob that consists of a single image of `1, 3, 224, 224` in `RGB` order. Before passing the image blob into the network, subtract RGB mean values as follows: [124, 117, 104]. In addition, values must be divided by 0.0167.
```

Comment:
should be removed or moved in description section

Reply:
Updated